### PR TITLE
Add container to job-properties schema

### PR DESCRIPTION
**Pull request by fschuch**
#### What
Add container to job-properties schema

#### Why
It is new in Slurm RESTD 0.0.38

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Review by jaimesouza — APPROVED** [review]

File: jobbergate-composed/docker-compose.yml [hunk 1]
```diff
@@ -156,7 +156,7 @@ services:
       retries: 5
 
   minio:
-    image: minio/minio
+    image: minio/minio:RELEASE.2023-10-25T06-33-25Z
```
Comment (fschuch) [inline jobbergate-composed/docker-compose.yml:159]: Task created to investigate and revert this change when possible: https://app.clickup.com/t/18022949/PENG-1829

**Comment by omnivector-qa** [comment]
The backport to `release/4.0` failed:
```
The process '/usr/bin/git' failed with exit code 1
```
To backport manually, run these commands in your terminal:
```bash
# Fetch latest updates from GitHub
git fetch
# Create a new working tree
git worktree add .worktrees/backport-release/4.0 release/4.0
# Navigate to the new working tree
cd .worktrees/backport-release/4.0
# Create a new branch
git switch --create backport-403-to-release/4.0
# Cherry-pick the merged commit of this pull request and resolve the conflicts
git cherry-pick -x --mainline 1 2c51a2ffacdb22499e609147171b2c827d763020
# Push it to GitHub
git push --set-upstream origin backport-403-to-release/4.0
# Go back to the original working tree
cd ../..
# Delete the working tree
git worktree remove .worktrees/backport-release/4.0
```
Then, create a pull request where the `base` branch is `release/4.0` and the `compare`/`head` branch is `backport-403-to-release/4.0`.

**Comment by omnivector-qa** [comment]
The backport to `release/4.0` failed:
```
The process '/usr/bin/git' failed with exit code 1
```
To backport manually, run these commands in your terminal:
```bash
# Fetch latest updates from GitHub
git fetch
# Create a new working tree
git worktree add .worktrees/backport-release/4.0 release/4.0
# Navigate to the new working tree
cd .worktrees/backport-release/4.0
# Create a new branch
git switch --create backport-403-to-release/4.0
# Cherry-pick the merged commit of this pull request and resolve the conflicts
git cherry-pick -x --mainline 1 2c51a2ffacdb22499e609147171b2c827d763020
# Push it to GitHub
git push --set-upstream origin backport-403-to-release/4.0
# Go back to the original working tree
cd ../..
# Delete the working tree
git worktree remove .worktrees/backport-release/4.0
```
Then, create a pull request where the `base` branch is `release/4.0` and the `compare`/`head` branch is `backport-403-to-release/4.0`.
